src/utils/formatPriceToTwoDecimals.ts

```diff
@@ -0,0 +1,3 @@
+export function formatToTwoDecimals(value: number): string {
+  return (Math.round(value * 100) / 100).toFixed(2)
```

Comment:
doesn't `Math.round(value * 100) / 100)` just do the job? No need for `toFixed(2)`.

Reply:
math round would round the number, but .toFixed(2) ensures that the decimal is always 2 places. so math round might result is .3 for example rather than .30

Reply:
Okay, word. I didn't know.

Reply:
same, trial and error